.github/workflows/nightly-build.yaml

```diff
@@ -3,6 +3,7 @@ name: Nightly
 on:
   schedule:
     - cron: '0 0 * * *' # Every day at midnight
+  workflow_dispatch:
```

Comment:
What this do?

Reply:
Oops, that wasn't meant to be included in this PR (and has already merged into master)...
It allows one to trigger the pipeline manually or via an API call tho.

Reply:
fixed :)